src/appshell/view/preferences/appearancepreferencesmodel.cpp

```diff
@@ -139,6 +139,17 @@ QStringList AppearancePreferencesModel::allFonts() const
     return uiConfiguration()->possibleFontFamilies();
 }
 
+ClipStyles::Style AppearancePreferencesModel::clipStyle() const
+{
+    return projectSceneConfiguration()->clipStyle();
+}
+
+void AppearancePreferencesModel::setClipStyle(ClipStyles::Style style)
+{
+    projectSceneConfiguration()->setClipStyle(style);
+    emit clipStyleChanged();
```

Comment:
1. add a check for value change so as not to send a change signal if there is no change
2. add connection to style change in init method and emit clipStyleChanged there